content/en/docs/releases/kubeflow-1.8.md

```diff
@@ -0,0 +1,278 @@
++++
+title = "Kubeflow 1.8"
+description = "Information about the Kubeflow 1.8 release"
+weight = 97
```

Comment:
Based on the [preview](https://deploy-preview-3615--competent-brattain-de2d6d.netlify.app/docs/releases/), it looks like 1.7 is on top of the 1.8. can you make sure the weight is adjusted to ensure 1.8 is on top?

Reply:
OOps, checking

Reply:
It is now solved thanks @annajung